2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,29 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+// PreviousPassword = ["fhD8!yrjj", "ttkTu.wer3", "dvyyeyY!5", "qwbfj76%", "tytT3729."];
+
+function validatePasswords(passwords) {
+  let   validity=[];
+  for ( let index=0; index < passwords.length; index++)
+  {
+    element=passwords[index];
```

Comment:
Good use of the other functions! What is wrong on this line of code?

Reply:
I think haven't defined element as a variable